### ISSUE TITLE
weston fails to start from tty

### ISSUE THREAD

**Issue by mviereck** [mviereck/x11docker#142]
On debian buster options `--weston` and `--weston-xwayland` fail to start from console with error message:
```
[13:52:45.514] initializing drm backend
[13:52:45.514] fatal: drm backend should be run using weston-launch binary, or your system should provide the logind D-Bus API.
[13:52:45.514] fatal: failed to create compositor backend
```
`weston-launch` fails with:
```
weston-launch: Permission denied. You should either:
 - enable systemd session support for weston-launch.
 - or add yourself to the 'weston-launch' group.
```

**Comment by mviereck** [comment]
Related debian bug report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=799325
The bug should be fixed in weston 5.0.0-3. Hopefully the update is published soon.

**Comment by mviereck** [comment]
Bug is fixed in weston 5.0.0-3
